### PR TITLE
Support balance query via environment

### PR DESCRIPTION
**Pull request by jjperezaguinaga**
- Adding balance query via environment
- See hoprnet/hoprnet#3351

### CODE REVIEW

**Review by robertkiel — APPROVED** [review]
lgtm, I guess we can set this PR to `ready for review`?

File: api/pages/api/balance/[environment]/[address]/hopr.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { providers, Wallet, utils, errors, Contract, BigNumber } from 'ethers';
```
Comment (tolbrino) [inline api/pages/api/balance/[environment]/[address]/hopr.ts:1]: Generally, the readme seems out of date now.

Also where is the `protocol-config.json` to be found and how is it synched with the original?

Comment (tolbrino) [reply api/pages/api/balance/[environment]/[address]/hopr.ts:1]: nvm the second point, found it

Comment (jjperezaguinaga) [reply api/pages/api/balance/[environment]/[address]/hopr.ts:1]: @tolbrino I want to clean up also old endpoints, but I rather do it in a separate PR. Happy to do it against this PR tho.

Comment (tolbrino) [reply api/pages/api/balance/[environment]/[address]/hopr.ts:1]: Just wanted to raise awareness. Can be done separately too.

**Comment by jjperezaguinaga** [comment]
@tolbrino @robertkiel this has to be merged by any of you ☺️

**Pull request merged**